### PR TITLE
Print usage recording

### PR DESCRIPTION
**Pull request by kenoir**

### CODE REVIEW

File: usage/app/model/UsageStatus.scala [hunk 1]
```diff
@@ -11,6 +11,8 @@ trait UsageStatus {
 }
 
 object UsageStatus {
+  implicit val reads =
```
Comment (jamesgorrie) [inline usage/app/model/UsageStatus.scala]: That'll break compilation no?

Comment (akash1810) [inline usage/app/model/UsageStatus.scala]: @jamesgorrie hence the WIP tag :wink:

Comment (kenoir) [inline usage/app/model/UsageStatus.scala]: Yup.

On Mon, 2 Nov 2015 at 18:01 Akash A <notifications@github.com> wrote:

> In usage/app/model/UsageStatus.scala
> <https://github.com/guardian/grid/pull/1418#discussion_r43658944>:
>
> > @@ -11,6 +11,8 @@ trait UsageStatus {
> >  }
> >
> >  object UsageStatus {
> > +  implicit val reads =
>
> @jamesgorrie <https://github.com/jamesgorrie> hence the WIP tag [image:
> :wink:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/grid/pull/1418/files#r43658944>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Comment by kenoir** [comment]
:+1:

**Pull request merged**